### PR TITLE
Import gouvfr templates

### PR DESCRIPTION
**Pull request by noirbizarre**
This pull-request reimport some gouvfr templates, factorize discussions template and fix discussions on reuses.

### CODE REVIEW

File: udata/templates/dataset/display.html [hunk 1]
```diff
@@ -31,7 +32,7 @@
         {{ _('Edit') }}
     </a>
 </div>
-{% endif %}
+{% endif%}
```
Comment (davidbgk) [inline udata/templates/dataset/display.html]: Why?

File: udata/templates/discussion/threads.html [hunk 2]
```diff
@@ -0,0 +1,67 @@
+{#
+    Discussion threads.
+    Expect the following context:
```
Comment (davidbgk) [inline udata/templates/discussion/threads.html]: :+1:

File: js/reuse/display.js [hunk 3]
```diff
@@ -1,16 +1,15 @@
...
-    'widgets/featured',
-    'widgets/follow-btn',
-    'widgets/issues-btn',
-    'widgets/share-btn',
-    'widgets/discussions-btn'
-], function(log) {
-    $(function() {
-        log.debug('Reuse display page');
-    });
+import 'widgets/featured';
```
Comment (davidbgk) [inline js/reuse/display.js]: `import featured from 'widgets'` would work here?

File: js/widgets/discussions-btn.js [hunk 4]
```diff
@@ -1,163 +1,168 @@
...
+}
+
+function submitNewComment(el, form, data) {
+    API.post(form.data('api-url'), data, function(data) {
+        var msg = i18n._('Your comment has been sent to the server');
+        Notify.success(msg);
+    }).error(function(e) {
+        var msg = i18n._('An error occured while submitting your comment');
+        Notify.error(msg);
+        console.error(e.responseJSON);
```
Comment (davidbgk) [inline js/widgets/discussions-btn.js]: Using `log.` is probably safer

File: js/widgets/discussions-btn.js [hunk 5]
```diff
@@ -1,163 +1,168 @@
...
+
+function hideNewForm(el, form) {
+    form.addClass('fadeOutDown');
+    setTimeout(function() {
+        form.addClass('hidden');
+    }, 1000);
+}
+
+function submitNewComment(el, form, data) {
+    API.post(form.data('api-url'), data, function(data) {
```
Comment (davidbgk) [inline js/widgets/discussions-btn.js]: ES6 arrow-functions everywhere?

File: js/widgets/discussions-btn.js [hunk 6]
```diff
@@ -1,163 +1,168 @@
...
-        }
+    if (!Auth.need_user(i18n._('You need to be logged in to add a discussion.')) || !$form.valid()) {
+        return false;
+    }
 
-        function displayNewCommentForm(el) {
-            var discussionId = el.data('discussion-id');
-            $('.list-group-form-' + discussionId).removeClass('hidden').addClass('fadeInDown');
-        }
+    log.debug('$form', $form);
```
Comment (davidbgk) [inline js/widgets/discussions-btn.js]: Do we want to keep those debugging logs? (same below)

File: less/udata/resource.less [hunk 7]
```diff
@@ -52,6 +53,19 @@
                 margin-top: 13px;
             }
         }
+
+        &.body-only {
+            margin-top: -@vspacing;
```
Comment (davidbgk) [inline less/udata/resource.less]: Mmh negative spacing is dangerous :)

File: udata/templates/reuse/display.html [hunk 8]
```diff
@@ -156,46 +156,9 @@ <h2 itemprop="name">{{ reuse.title }}</h2>
...
-                                {{ _('Unfollow') }}
-                                {% else %}
-                                title="{{ _("I'll be informed about this organization news") }}">
-                                <span class="glyphicon glyphicon-eye-open"></span>
-                                {{ _('Follow') }}
-                                {% endif %}
-                            </button>
-                        </div>
+                        {% with organization=reuse.organization %}
+                        {% include theme('organization/sidebar-producer.html') %}
```
Comment (davidbgk) [inline udata/templates/reuse/display.html]: Indent?

File: udata/templates/dataset/display.html [hunk 9]
```diff
@@ -42,238 +43,450 @@
...
+            {# Right sidebar #}
+            <aside class="col-md-3 col-sm-3">
+
+                {# Optionnal organization panel #}
+                {% if dataset.organization %}
+                <div class="panel panel-default text-center">
+                    <div class="panel-body">
+                        <h3 class="text-left">{{ _('Producer') }}</h3>
+                        {% with organization=dataset.organization %}
+                        {% include theme('organization/sidebar-producer.html') %}
```
Comment (davidbgk) [inline udata/templates/dataset/display.html]: Indent?

**Comment by davidbgk** [comment]
Impressive work, r+wc (+ integration tests? ;) )

File: udata/templates/dataset/display.html (hunk 1 above)
Comment (noirbizarre) [inline udata/templates/dataset/display.html]: Not meant, I will remove it.

File: udata/templates/discussion/threads.html (hunk 2 above)
Comment (noirbizarre) [inline udata/templates/discussion/threads.html]: Yep, explicit is better than implicit ;)

File: js/widgets/discussions-btn.js (hunk 6 above)
Comment (noirbizarre) [inline js/widgets/discussions-btn.js]: Nop, I will remove it

File: js/widgets/discussions-btn.js (hunk 4 above)
Comment (noirbizarre) [inline js/widgets/discussions-btn.js]: Yep, you're right

File: js/reuse/display.js (hunk 3 above)
Comment (noirbizarre) [inline js/reuse/display.js]: I don't think so. This syntax is meant to import the default export from a given file.

File: js/widgets/discussions-btn.js (hunk 5 above)
Comment (noirbizarre) [inline js/widgets/discussions-btn.js]: I will too

File: less/udata/resource.less (hunk 7 above)
Comment (noirbizarre) [inline less/udata/resource.less]: If you have a better solution wihtout breaking the whole style, I take it.
I think we should cleanup these styles.

File: js/widgets/discussions-btn.js (hunk 5 above)
Comment (noirbizarre) [inline js/widgets/discussions-btn.js]: In fact not needed beacause there is no reference to this.

**Comment by noirbizarre** [comment]
I will cover all thoses screens with integration tests next week (It will be a good exercise to learn)

**Pull request merged**